addons/project/static/src/project_sharing/project_sharing.js

```diff
@@ -34,6 +34,9 @@ export class ProjectSharingWebClient extends Component {
 
     async _showView() {
         const { action_name, project_id, open_task_action } = session;
+        if (action_name.help) {
+            action_name.help = markup(action_name.help);
```

Comment:
I think we also have to escape the `help` content to be sure it is safe.

```suggestion
            action_name.help = markup(escape(action_name.help));
```

To import `escape` function we have to do this:
```js
import { escape } from "@web/core/utils/strings";
```

Reply:
Done! Thanks :slightly_smiling_face: